src/app.py

```diff
@@ -323,6 +337,10 @@ def upload_csv():
     data["transfer_source"] = ""
     data["transfer_destination"] = ""
 
+    data["sealed_state"] = data["sealed_state"].map(
```

Comment:
Does `sealed_state` exist in the incoming csv? If it doesn't exists, how will this be handled?

Reply:
yes, isSealed should be part of uploading CSV. I will check with Dimuthu if it is a mandatory column to send in CSV. 
If it's mandatory then I will include a check to ensure it's not empty.

Reply:
That makes sense. Let's do a sanity check in CSV itself. If the column doesn't exist, we can add a default one?

Reply:
Making the IsSealed column a mandatory column and value in the CSV. 
Added the checks in the code as well. 
@NISH1001

Reply:
@ub0005 Eh thanks... I saw the code. The checks should work...